research/open_aq_search.py

```diff
@@ -0,0 +1,99 @@
+"""
+Testing the OpenAQ API to use a list of airport locations
+to search and find air quality measurements within a particular
+distance from those locations.
+"""
+
+import csv
+import pandas as pd
+import requests
+
+OPEN_AQ_API_KEY = "d568965fba34916d82780a672e32d0029f0e45f5293dc4f45a1ae33fe827623b"
+
+# -------------------- CONFIG --------------------
+radius = "10000"          # Search radius [m]
+loc_result_limit = "1"   # Max number of location results returned
+measurement_result_limit = "10"  # max number of measurement results returned for a location (for some reason <=2 causes a 402 timeout)
+# ------------------------------------------------
+
+def long_lat_conv(degrees: int, minutes: int, seconds: int, direction: str) -> float:
+    """
+    Convert longitude or latitude from degrees, minutes, seconds, and direction to 
+    a decimal value.
+    
+    Return: float
+
+    """
+    dd = float(degrees) + float(minutes)/60 + float(seconds)/(60*60)
+    if direction == 'W' or direction == 'S':
+        dd *= -1
+    return dd
+
+
+# Import list of airports and their lat/long locations
+
+# read text file into pandas DataFrame
+# airports = pd.read_csv("../data/GlobalAirportDatabase.txt", sep=",")
+# print(airports)
+
+with open("../data/GlobalAirportDatabase.txt", newline="") as f:
+    reader = csv.reader(f)
+    data = list(reader)
+
+print(data[1])
+
+airports_list = []
+
+for index in range(len(data)):
+    curr_airport = data[index][0].split(":")
+    curr_name = curr_airport[2]
+    if curr_name != "N/A":
+        curr_lat = long_lat_conv(int(curr_airport[5]), int(curr_airport[6]), int(curr_airport[7]), curr_airport[8])
+        curr_long = long_lat_conv(int(curr_airport[9]), int(curr_airport[10]), int(curr_airport[11]), curr_airport[12])
+        print(str(round((index / len(data)) * 100, 1)) + "%", curr_name, curr_lat, curr_long, flush=True)
```

Comment:
This print statement is very oddly formatted - I'm not sure exactly what this is going to print?